apps/tasks/lib/pages/settings_page.dart

```diff
@@ -0,0 +1,66 @@
+
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/constants.dart';
+
+class SettingsPage extends StatefulWidget{
+  const SettingsPage({super.key});
```

Comment:
See #217, should now be called `SettingsScreen`

Please also add a description like
```
/// Screen for settings and other app options
```